### DISCUSSION TITLE
Does this work concurrently with Libre app

### DISCUSSION THREAD

**Discussion by hgh1001** [creepymonster/GlucoseDirect#432] [Q&A]
Very interested in this but also want to still use the Libre 3 app. Can I scan with both apps so readings go to both places? 

Thanks!

**Comment by creepymonster** [comment]
Yes, it is absolutely needed that the Libre 3 app runs and reads the data, which is then transferred to LibreLinkUp and read there by GD.

The first step is that LibreLinkUp must be set up.

https://youtube.com/shorts/Ljj-eQ4Jl30

Then set up GD.

https://youtube.com/shorts/Xn5jM2rJcb0

Many greetings
Reimar

**Reply by hgh1001** [reply]
Thanks. I asked because of the following step. If you are already scanning with the Libre app and it is reading app to app help me understand the need for this scanning step: 

In "Glucose Direct" on the "Overview" select "Scan Sensor" and scan the Libre 3 sensor on the arm with it. Attention, the scanning process takes longer than with the Libre 2 - it can take 5-8 seconds until the sensor is scanned. While scanning, hold the phone as still as possible over the sensor - you will hear a beep when the scan is complete.

**Reply by hgh1001** [reply]
Any explanation on why we need to scan the sensor? Thanks!

**Reply by hgh1001** [reply]
Thanks. Much more info than I was looking for but looks like the bottom line is to get information so you can read the data from the app? If so that makes sense.

**Reply by creepymonster** [reply]
> Any explanation on why we need to scan the sensor? Thanks!

The scan of the sensor is needed so that I can load the correct "connector".

The Libre 2 connects to the sensor via Bluetooth and decodes its data.

With the Libre 3, GD also connects to the sensor, but can not yet do anything with the data that the sensor transmits. I use the Bluetooth connection that the Libre 3 app establishes only to be woken up every minute by iOS (when the Libre 3 app receives the data from the sensor) and then to retrieve the data via LibreLinkUp. Without a connection to a Bluetooth device (which regularly reports), iOS otherwise very quickly terminates an app in the background - the Bluetooth connection ensures that this does not happen.

Translated with www.DeepL.com/Translator (free version)

**Reply by Kaltkalt** [reply]
Thanks for the explanation I was wondering about this too.  And I love the app, I think it’s commercial negligence for the Libre3 app to not have Apple Watch support.  Almost defeats the point unless you’re one of those people who can’t stand pricking your finger.  It’s not that bad and after years I can’t imagine anyone would still be a wimp about it and I’m a pain pussy myself.  And we all know Apple no longer innovates (no need, at least so they think), so the Apple Watch will *never* have noninvasive blood glucose monitoring like has been rumored to always be coming in the next version.  Like fusion and a "climate change" doomsday, it’s perpetually ten years away.  So I’m sincerely very grateful for GD.  Thank you!
